simulate-event.js

```diff
@@ -281,6 +304,20 @@ module.exports = function (element, type, options) {
 
   event = extend(document.createEvent(eventType), options);
 
+  // Handle differences between `initKeyboardEvent` and `initKeyEvent`.
+  if (initEvent === 'initKeyboardEvent') {
+    if (event[initEvent] === void 0) {
+      initEvent = 'initKeyEvent';
+    } else if (!('modifiersList' in options)) {
```

Comment:
Should this be an `else if`? Just checking, but if the init event is changed do we want to skip adding the modifier list still?